cmd/podman/restart.go

```
@@ -114,13 +145,49 @@ func restartCmd(c *cli.Context) error {
 		})
 	}
 
-	maxWorkers := shared.Parallelize("restart")
-	if c.GlobalIsSet("max-workers") {
-		maxWorkers = c.GlobalInt("max-workers")
-	}
-
-	logrus.Debugf("Setting maximum workers to %d", maxWorkers)
-
 	restartErrors, errCount := shared.ParallelExecuteWorkerPool(maxWorkers, restartFuncs)
 	return printParallelOutput(restartErrors, errCount)
 }
+
+func stopRootlessContainers(stopContainers []*libpod.Container, timeout uint, useTimeout bool, maxWorkers int) error {
+	var stopFuncs []shared.ParallelWorkerInput
+	for _, ctr := range stopContainers {
+		state, err := ctr.State()
+		if err != nil {
```

Should we just report this error and continue?